src/homework2/task7.py

```diff
@@ -0,0 +1,87 @@
+import os
+
+'''https://www.hackerrank.com/challenges/py-if-else/problem
+'''
+
+n = int(input('Enter your number\n'))
+if n % 2 != 0:
+    print("Weird")
+elif 2 <= n <= 5:
+    print("Not Weird")
+elif 6 <= n <= 20:
+    print("Weird")
+elif n > 20:
+    print("Not Weird")
+
```

Comment:
why is docstring separated from the previous task with 1 blank line and from it's task with 2?